packages/grafana-ui/src/components/Icon/Icon.tsx

```diff
@@ -1,11 +1,18 @@
-import React from 'react';
```

Comment:
@dprokop @torkelo Could you look at this new Icon implementation and let me know what do you think?

I have decided to currently import **react-unicons package** and whenever I replace previous icon with the new one, I add it to the icon types. This way, we will have a list of all icons and we can then decide wether we want to keep  **react-unicons package**  or wether we want to just import the icons that we are going to use. 

I have added 1 monochrome icon, to test out that. implementation works for both options.

Reply:
Also, do we do [tree shaking](https://webpack.js.org/guides/tree-shaking/) for grafana/ui? I suspect that no, because I didn't find it in webpack, but maybe I wasn't looking at the right place. Just a thing that I am considering would be useful,  if we decided to use react-unicons package.